src/encoded/schemas/file_other.json

```diff
@@ -0,0 +1,41 @@
+{
+    "title": "Other file",
+    "description": "Other files to ingest.",
```

Comment:
This should not say "ingest". This is a holdover from when it was FileIngest. To be properly-general purpose, this should just be "Files of a kind other than the ones we've assigned a task-specific File type."